webrender/src/platform/macos/font.rs

```diff
@@ -69,23 +70,47 @@ fn supports_subpixel_aa() -> bool {
     data[0] != data[1] || data[1] != data[2]
 }
 
-fn get_glyph_metrics(ct_font: &CTFont, glyph: CGGlyph) -> GlyphMetrics {
+fn get_glyph_metrics(ct_font: &CTFont,
+                     glyph: CGGlyph,
+                     x_subpixel: SubpixelOffset,
```

Comment:
I was actually thinking of rewriting this and passing in GlyphKeys instead to both get_glyph_metrics and rasterize_glyphs. What do you think about that?